specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -3294,6 +4788,10 @@
         "password": {
           "type": "string"
         },
+        "securityToken": {
```

Comment:
>securityToken [](http://example.com/codeflow?start=9&length=13)

If this is a secret, it's not allowed in GET payloads and must be tagged with x-ms-secret. This type is currently present in GET payloads.

Reply:
marked as x-ms-secret already